spyder/plugins/completion/kite/utils/status.py

```diff
@@ -16,6 +16,10 @@
 # Third-party imports
 import psutil
 
+NOT_INSTALLED = 'not installed'
+RUNNING = 'running'
```

Comment:
Change this to
```python
RUNNING = 'ready'
```
This will prevent the status indicator from flickering between `running` and `ready` as the user clicks around the editor UI.

Reply:
(@its-dhung we should take note of this on our side so that we know changing the short text to something other than `"ready"` in the Kite status response may end up causing flickering on Spyder)

Reply:
Will make a note of this in the API doc.